Framework/MDAlgorithms/test/CreateMDTest.h

```diff
@@ -250,8 +250,5 @@ class CreateMDTest : public CxxTest::TestSuite {
         "__CreateMDTest_mdworkspace");
     Mantid::API::AnalysisDataService::Instance().remove(
         "__CreateMDTest_mdworkspace_fb");
-
-    if (Poco::File(filename).exists())
-      Poco::File(filename).remove();
```

Comment:
Why do we no longer remove this workspace?

Reply:
The file you mean? Ah this is because if it removed in the destructor of the File Backed MD workspace that is deleted on line 251.

What I was getting on windows was a race condition where the file still existed for the exists check but had gone before the remove call, which then threw an exception.

The file is still gone.  This was always belt and braces code (which is the exact comment on those lines if you go into git blame).

Reply:
Ah okay, and as a bonus this potentially removes a race condition in this code which could have failed tests at random, good spot :+1: